BothamUI.xcodeproj/project.pbxproj

```diff
@@ -14,16 +14,16 @@
 		6B6CCD341BDD0FFB0015A4D4 /* BothamUI.h in Headers */ = {isa = PBXBuildFile; fileRef = 6B6CCD331BDD0FFB0015A4D4 /* BothamUI.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		6B6CCD511BDD11810015A4D4 /* BothamViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B6CCD501BDD11810015A4D4 /* BothamViewController.swift */; };
-		6B6CCD531BDD12C40015A4D4 /* Presenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B6CCD521BDD12C40015A4D4 /* Presenter.swift */; };
+		6B6CCD531BDD12C40015A4D4 /* BothamPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B6CCD521BDD12C40015A4D4 /* BothamPresenter.swift */; };
 		6B82AD471C075A16009247FB /* BothamPullToRefreshHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B82AD461C075A16009247FB /* BothamPullToRefreshHandler.swift */; };
 		6B82AD491C075A2C009247FB /* BothamPullToRefreshPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B82AD481C075A2C009247FB /* BothamPullToRefreshPresenter.swift */; };
 		6B82AD4D1C077A0E009247FB /* BothamPullToResfreshUI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B82AD4C1C077A0E009247FB /* BothamPullToResfreshUI.swift */; };
 		6B82AD4F1C0780E9009247FB /* BothamPullToRefresh.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B82AD4E1C0780E9009247FB /* BothamPullToRefresh.swift */; };
 		6B8E73281C101D240006F9E1 /* BothamViewDataSource.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73271C101D240006F9E1 /* BothamViewDataSource.swift */; };
-		6B8E732A1C101D8F0006F9E1 /* NavigatorContainer.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73291C101D8F0006F9E1 /* NavigatorContainer.swift */; };
-		6B8E732C1C101DBA0006F9E1 /* NavigationPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E732B1C101DBA0006F9E1 /* NavigationPresenter.swift */; };
+		6B8E732A1C101D8F0006F9E1 /* BothamNavigatorContainer.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73291C101D8F0006F9E1 /* BothamNavigatorContainer.swift */; };
+		6B8E732C1C101DBA0006F9E1 /* BothamNavigationPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E732B1C101DBA0006F9E1 /* BothamNavigationPresenter.swift */; };
 		6B8E732F1C101E570006F9E1 /* TableViewNavigationDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E732E1C101E570006F9E1 /* TableViewNavigationDelegate.swift */; };
-		6B8E73311C101FA90006F9E1 /* Storyboard.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73301C101FA90006F9E1 /* Storyboard.swift */; };
+		6B8E73311C101FA90006F9E1 /* BothamStoryboard.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73301C101FA90006F9E1 /* BothamStoryboard.swift */; };
 		6B8E73371C119C800006F9E1 /* UINavigationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6B8E73361C119C800006F9E1 /* UINavigationController.swift */; };
```

Comment:
Sorry :( still missing TableViewNavigationDelegate

And should we rename LoadingView to BothamLoadingView, to avoid problem with project that already have a LoadingView ?